From aade9b70aabd8a97dd8a28cda2cf4d0694dd7350 Mon Sep 17 00:00:00 2001
From: Behdad Esfahbod <behdad@behdad.org>
Date: Tue, 13 Aug 2019 16:09:20 -0700
Subject: [PATCH] [pool] Fix alignment assertion

I *think* it should fix https://github.com/harfbuzz/harfbuzz/issues/1901

Ie. if on a system, alignof(void*) < sizeof(void*)...

Downloaded from upstream commit
https://github.com/harfbuzz/harfbuzz/commit/aade9b70aabd8a97dd8a28cda2cf4d0694dd7350

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 src/hb-pool.hh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hb-pool.hh b/src/hb-pool.hh
index ff0ee194c..2dd84968e 100644
--- a/src/hb-pool.hh
+++ b/src/hb-pool.hh
@@ -77,7 +77,7 @@ struct hb_pool_t
 
   static_assert (ChunkLen > 1, "");
   static_assert (sizeof (T) >= sizeof (void *), "");
-  static_assert (alignof (T) % sizeof (void *) == 0, "");
+  static_assert (alignof (T) % alignof (void *) == 0, "");
 
   struct chunk_t
   {
